### DISCUSSION TITLE
Add a Currency field type

### DISCUSSION THREAD

**Discussion by zackkatz** [UseDataKit/SDK#22] [Ideas]
Number formatting is one thing…currency is another!

Currencies need to handle different challenges:

- **Numerical sorting**
- Symbol handling
  - Symbol before or after
  - Space between symbol and price or not "£ 100" vs "$100"
- Decimal handling
- Thousand separator

**Comment by mrcasual** [comment]
A good reading: https://gist.github.com/rgs/6509585
